test/fixtures/index.test.ts

```diff
@@ -111,6 +111,14 @@ describe('buildDepGraph', () => {
     expect(actualGraph).toBeDefined();
     expect(actualGraph.getDepPkgs().length).toBe(1);
   });
+
+  it('on fixture with conflicting python declarations yields graph successfully', () => {
+    jest.spyOn(console, 'warn').mockImplementation();
```

Comment:
any reason to mock this implementation?

Reply:
Sorry, see my comment on the other use of this: 
> // Mock impl to prevent polluting logs in tests
I'll update this snippet with this comment too or do you think the mock implementation is kind of pointless?